osu.Game/Screens/Edit/Compose/Components/BlueprintContainer.cs

```diff
@@ -441,7 +483,11 @@ protected virtual void OnBlueprintDeselected(SelectionBlueprint<T> blueprint)
 
         private Vector2[][] movementBlueprintsOriginalPositions;
         private SelectionBlueprint<T>[] movementBlueprints;
-        private bool isDraggingBlueprint;
+
+        /// <summary>
+        /// Whether a blueprint is currently being dragged.
+        /// </summary>
+        private bool isDraggingBlueprint { get; set; }
```

Comment:
This is a huge nit but I find it a bit strange that this became a property while `selectedBlueprintAlreadySelectedOnMouseDown` and `clickSelectionHandled` are still fields.

Reply:
This is a left-over from when I tried to implement the cyclic selection in `SkinBlueprintContainer` (where this became `protected`). I'll revert the change.